modules/service/src/main/scala/lucuma/odb/graphql/mapping/TargetEnvironmentMapping.scala

```diff
@@ -116,6 +133,11 @@ trait TargetEnvironmentMapping[F[_]: Temporal]
       end   <- Elab.liftR(rEnd)
       env   <- Elab.env(AvailabilityStartParam -> start, AvailabilityEndParam -> end)
     } yield env
+
+    case (TargetEnvironmentType, "blindOffsetTarget", Nil) =>
```

Comment:
Do we even need an elaborator in this case?  I would think it would work without one.